### PR TITLE
Android NDK agent v1.0.2 release notes(Target to release: 5/4/23)

### PR DESCRIPTION
**Pull request by ywang-nr**
## Give us some context

* What problems does this PR solve?
* 1. Kotlin library has vulnerability in certain versions, reference [here](https://security.snyk.io/package/maven/org.jetbrains.kotlin:kotlin-stdlib)
* 2. This release is simply updating to a version that has addressed the vulnerability
* 3. Target to release 5/4/23

### CODE REVIEW

**Comment by ywang-nr** [comment]
@ally-sassman @cthomas-newrelic for review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-102.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+subject: Android agent
+title: Android NDK agent v1.0.2
+releaseDate: '2023-05-04'
+version: 1.0.2
+---
+
+## Fixed
+
+* Updated kotlin library to a version that addressed vulnerability
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-102.mdx:10]:
```suggestion
* Updated Kotlin library to a version that addressed a vulnerability.
```

**Review by ally-sassman — APPROVED** [review]

**Comment by ally-sassman** [comment]
Hi @ywang-nr can you please resign the CLA agreement?

**Comment by ywang-nr** [comment]
> Hi @ywang-nr can you please resign the CLA agreement?

Done

**Comment by ally-sassman** [comment]
@ywang-nr Thanks! Can I merge?

**Comment by ally-sassman** [comment]
@ywang-nr Oops sorry, just seeing your response now. I reverted it and will wait to hear from you to merge.

**Comment by ywang-nr** [comment]
> @ywang-nr Oops sorry, just seeing your response now. I reverted it and will wait to hear from you to merge.

@ally-sassman  We can merge now, thank you!

**Comment by ally-sassman** [comment]
@ywang-nr Merged: https://github.com/newrelic/docs-website/pull/12919

**Pull request merged**